src/components/mainContent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+<v-card id="contentBox" fill-height>
+  <v-card-title>
+    <v-text-field
+    v-model="search"
+    append-icon="mdi-magnify"
+    label="Search"
+    single-line
+    hide-details
```

Comment:
make sure this is properly indented so that it won't cause confusion if we have a long HTML file